package/src/lib/SpotlightTour.context.ts

```diff
@@ -97,15 +74,13 @@ export interface StopParams {
   isLast: boolean;
 }
 
+export interface FloatingProps {
+  middleware?: Middleware[];
+  placement?: Placement;
+  sameScrollView?: boolean;
+}
```

Comment:
Missing docs on the interface and each property. Here you can also document what's the default value of each property (if any)